pkgs/applications/misc/terminal-colors/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, lib, help2man, python3, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "terminal-colors";
+  version = "3.0.1";
+  outputs = [ "out" "man" ];
+
+  src = fetchFromGitHub {
+    owner = "eikenb";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-hekt77/FhSTMEARVuck49/Q1dIuqkwbOYmgGD1IItyc=";
+  };
+
+  buildInputs = [ python3 ];
+  nativeBuildInputs = [ help2man ];
+
+  postPatch =
+    # This sed command modifies output of --version command in way that
+    # makes manpage generated by help2man(1) prettier.
+    ''
+      sed -r -i "3s/([0-9.]+)/$pname - \1\\n/" ./$pname
+    ''
+    # Upstream shebang uses /usr/bin/env, which is not present in Nix
+    # sandbox, so we need to patch it before running help2man, otherwise
+    # it would fail with "no such file or directory".
+    + ''
+      patchShebangs .
```

Comment:
Can we be more specific which files need patching?

Reply:
Fixed.

Reply:
You didn't push it.

Reply:
```
    # Upstream shebang of "terminal-colors" python script uses
    # /usr/bin/env, which is not present in Nix sandbox, so we need to
    # patch it before running help2man, otherwise it would fail with "no
    # such file or directory".
```

I think this is quite specific about what is being patched and why.